pkg/cmd/pulumi/destroy.go

```diff
@@ -207,6 +208,10 @@ func newDestroyCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVarP(
 		&refresh, "refresh", "r", false,
 		"Refresh the state of the stack's resources before this update")
+	cmd.PersistentFlags().BoolVar(
+		&skipRefresh, "skip-refresh", false,
+		"Skip the refreshing of the state of the stack's resources before this update. Typically used "+
+			"when overriding a Pulumi project setting")
```

Comment:
I wonder if we could piggyback off of the existing `refresh` flag here. Is there any way to differentiate between a flag that is unset and therefore has its default value and a flag that is set but has the same value as its default? That would allow us to use `--refresh=false` rather than `--skip-refresh`.

Reply:
I am happy to change this of course but `--refresh=false` doesn't feel as nice to me as `--skip-refresh` :) But 🤷

Reply:
Agree that `skip-refresh` is "clearer" but since we already have a bool flag for `refresh` and passing `refresh=false` is already a valid use of the flag let's stick to that. Otherwise we'll kind of have 2 ways of specifying "don't refresh"